nft_ingester/src/program_transformers/token_metadata/v1_asset.rs

```diff
@@ -296,25 +296,33 @@ pub async fn save_v1_asset<T: ConnectionTrait + TransactionTrait>(
             txn.execute(query).await?;
         }
     }
-    txn.commit().await?;
-    let creators = data.creators.unwrap_or_default();
-    if !creators.is_empty() {
-        let mut creators_set = HashSet::new();
-        let existing_creators: Vec<asset_creators::Model> = asset_creators::Entity::find()
+
+    // checks if the creator to index is outdated. This assumes that all creator rows with same AssetId have the same SlotUpdated
+    let creator_outdated = asset::Entity::find()
```

Comment:
I would call this `should_update_creators` and clarify that we only update creators when indexing a newer update for the given asset.